docs/content/installation/csi_ug_upgrade.md

```diff
@@ -0,0 +1,17 @@
+# Upgrading the CSI driver
+
+Use this information to upgrade the IBM® block storage CSI driver.
+
+- The OpenShift web console and OperatorHub.io both automatically upgrade the the CSI (Container Storage Interface) driver when a new version is released.
```

Comment:
double `the`

Reply:
done

Reply:
note that this isn't the latest branch for 1.7.0

Reply:
I know, it wasn't in the 1.7.0 PR diff, so I couldn't comment on it there